pkg/olm/subscription.go

```diff
@@ -130,8 +132,8 @@ func OperatorCleanup(cs *clients.Clients, name string) {
 		testsuit.T.Errorf("failed to delete CSVs in namespace %s \n %v", OperatorsNamespace, err)
 	}
 
-	log.Printf("Output %s \n", cmd.MustSucceed(
-		"oc", "delete", "-n", OperatorsNamespace,
+	log.Printf("Output %s \n", cmd.Run(
```

Comment:
1. you can use `--ignore-not-found` (without `=true`)
2. command returns 0 (success) when you have `--ignore-not-found` so IMHO you can leave `MustSucceed`, can't you?